src/lib/supabase.ts

```diff
@@ -0,0 +1,7 @@
+import { createClient } from "@supabase/supabase-js";
+import { env } from "../../env";
+
+const supabaseURL = env.SUPABASE_URL;
```

Comment:
no need for these variables, we can just use the values directly

Reply:
ok